### DISCUSSION TITLE
What does this do?

### DISCUSSION THREAD

**Discussion by dhruvmaindola227** [vincenzopalazzo/material-ui-swing#192] [First contribution]
Hi. Im very new to open source and i want to start contributing. I forked and opened this project in my IntelliJ but im unable to understand how to run this exactly or what to do? Can anyone please help me out with this? If im able to understand i'll be glad to contribute in any way i can.

**Comment by vincenzopalazzo** [comment]
Sorry for the delay here and sorry if you have not found good information in the repository to solve your issue, I added a guide on how to contribute in the repository https://github.com/vincenzopalazzo/material-ui-swing/blob/master/docs/dev/MAINTAINERS.md

However, if you are opening an issue in this repository you should be able to answer alone to this the following question in your issue 
>What does this do?

BTW you can start from there https://github.com/vincenzopalazzo/material-ui-swing/wiki

**Comment by dhruvmaindola227** [comment]
Thanks a lot!
